src/tests/ftest/util/command_utils.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, namespace, command, path="", subprocess=False, check_results=
         # used to check on the progress or terminate the command.
         self._exe_names = [self.command]
 
+        # If set use the full command string when returning the 'command_regex' property
+        self.full_command_regex = False
```

Comment:
Nit: since `command_regex` already exists and is a string, `full_command_regex` implies this is some sort of "full" version of `command_regex`. It's more verbose but I'd recommend something like `use_full_command_regex` or similar so the variable name reflects that it's a bool and not a string / "full" version

Reply:
I do prefer `use_full_command_regex` in this context.